src/pre_process/m_checker.fpp

```diff
@@ -197,4 +198,51 @@ contains
 
     end subroutine s_check_inputs_misc
 
+    subroutine s_check_bc
+
+        integer :: i
+
+        @:PROHIBIT(num_bc_patches > num_bc_patches_max, "num_bc_patches must be <= 10")
```

Comment:
Rewrite to remove the hardcoded number 10. There's a routine called `s_int_to_str` in `m_helper.fpp` that can be used for this. There should be an example somewhere else in the checker routines on how to do this.